### DISCUSSION TITLE
Sidebar navigation bug

### DISCUSSION THREAD

**Discussion by karoltonik** [twilio-labs/paste#3480] [Help: Figma] [answered]
Hey, noticed a tiny bug in the Segment sidebar navigation; the 'Connections' button has the wrong icon.

<img width="518" alt="image" src="https://github.com/twilio-labs/paste/assets/141224835/b33f7788-2354-40de-ac35-eba54b72bca6">

Regards, kd

**Comment by serifluous** [comment, marked as answer]
Good catch, thank you! I just published the fix.

By the way, if you ever want to open a branch to submit a fix, too, we'd definitely welcome it.
